<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="alter1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>322</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="alter2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="alter3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>440</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="alter4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>500</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="proxPerg">
    <property name="geometry">
     <rect>
      <x>674</x>
      <y>570</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="perguntaWidget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>371</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="alternativaWidget" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>381</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>460</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>520</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>580</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="alter5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>560</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>alter1</zorder>
   <zorder>alter2</zorder>
   <zorder>alter3</zorder>
   <zorder>alter4</zorder>
   <zorder>proxPerg</zorder>
   <zorder>perguntaWidget</zorder>
   <zorder>alternativaWidget</zorder>
   <zorder>radioButton</zorder>
   <zorder>radioButton_4</zorder>
   <zorder>radioButton_3</zorder>
   <zorder>radioButton_2</zorder>
   <zorder>radioButton_5</zorder>
   <zorder>alter5</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
